src/languageHandler.tsx

```diff
@@ -346,6 +346,20 @@ export function setLanguage(preferredLangs: string | string[]) {
     });
 }
 
+export function setSpellCheckLanguages(preferredLangs: string[]) {
```

Comment:
What's the reason for these functions, as opposed to accessing `PlatformPeg` directly?